### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Or-Geva**
- [ ] All [tests](https://github.com/jfrog/jfrog-vscode-extension#building-and-testing-the-sources) passed. If this feature is not already covered by the tests, I added new tests.
- [ ] I used `npm run format` for formatting the code before submitting the pull request.
-----

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -372,6 +372,10 @@ If however your policies are referenced through an Xray Watch or Watches, follow
 By default, paths containing the words `test`, `venv` and `node_modules` are excluded from Xray scan.
 The exclude pattern can be configured in the [Extension Settings](#extension-settings).
 
+### Exclude development dependencies during the scan
```
Comment (attiasas) [inline README.md]: I think we should also add it in the table of content, what do you think?

**Review by attiasas — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -372,6 +371,10 @@ If however your policies are referenced through an Xray Watch or Watches, follow
 By default, paths containing the words `test`, `venv` and `node_modules` are excluded from Xray scan.
 The exclude pattern can be configured in the [Extension Settings](#extension-settings).
 
+### Exclude Development Dependencies During Scan
```
Comment (yahavi) [inline README.md:374]: Please add it to the table of contents

File: README.md [hunk 3]
```diff
@@ -372,6 +371,10 @@ If however your policies are referenced through an Xray Watch or Watches, follow
...
+
+Development dependencies are scanned by default. You can skip it by choosing `Exclude Dev Dependencies` under Settings.
```
Comment (yahavi) [inline README.md:376]:
```suggestion
Development dependencies are scanned by default. You can exclude them by choosing `Exclude Dev Dependencies` in the [Extension Settings](#extension-settings).
```

**Review by yahavi — CHANGES_REQUESTED** [review]
Let's also add `Exclude dev dependencies` in the Features table under npm.

**Review by yahavi — APPROVED** [review]

**Pull request merged**